app/components/publish-actions.tsx

```diff
@@ -149,9 +160,23 @@ const Embed = ({ chartWrapperRef, configKey, locale }: PublishActionProps) => {
                   "For embedding visualizations in systems without JavaScript support (e.g. WordPress).",
               })}
             />
+            <EmbedToggleSwitch
+              value="without-border"
+              checked={withoutBorder}
+              onChange={(_, checked) => handleStylingChange(checked)}
+              label={t({
+                id: "publication.embed.iframe.remove-border",
+                message: "Iframe without border",
+              })}
+              infoMessage={t({
+                id: "publication.embed.iframe.remove-border.warn",
+                message:
+                  "For embedding visualizations in systems without a Border.",
+              })}
+            />
           </Flex>
           <CopyToClipboardTextInput
-            content={`<iframe src="${embedUrl}" width="100%" style="${isResponsive ? "" : `height: ${iframeHeight || 640}px; `}border:0px #ffffff none;" name="visualize.admin.ch"></iframe>${isResponsive ? `<script type="text/javascript">!function(){window.addEventListener("message", function (e) { if (e.data.type === "${CHART_RESIZE_EVENT_TYPE}") { document.querySelectorAll("iframe").forEach((iframe) => { if (iframe.contentWindow === e.source) { iframe.style.height = e.data.height + "px"; } }); } })}();</script>` : ""}`}
+            content={`<iframe src="${embedUrl}" width="100%" style="${isResponsive ? "" : `height: ${iframeHeight || 640}px; `}border: ${withoutBorder ? "0px #ffffff none;" : "1px #ffffff;"}" name="visualize.admin.ch"></iframe>${isResponsive ? `<script type="text/javascript">!function(){window.addEventListener("message", function (e) { if (e.data.type === "${CHART_RESIZE_EVENT_TYPE}") { document.querySelectorAll("iframe").forEach((iframe) => { if (iframe.contentWindow === e.source) { iframe.style.height = e.data.height + "px"; } }); } })}();</script>` : ""}`}
```

Comment:
Here we don't need to modify this, only `disableBorder=true` URL param is needed (iframe itself needs to not have a border at all times).